### PR TITLE
fix: remove delete tags for cell_table and meta_table

### PR DESCRIPTION
**Pull request by jemmyshin**
this pr remove delete flags for cell_table and meta_table

### CODE REVIEW

File: annlite/storage/table.py [hunk 1]
```diff
@@ -435,29 +440,33 @@ def get_latest_commit(self):
...
     def delete_address(self, doc_id: str, commit: bool = True):
-        sql = f'UPDATE {self.name} SET _deleted = 1, time_at = ? WHERE _doc_id = ?'
-        self._conn.execute(
-            sql,
-            (
-                time_now(),
-                doc_id,
-            ),
-        )
+        # sql = f'UPDATE {self.name} SET _deleted = 1, time_at = ? WHERE _doc_id = ?'
```
Comment (numb3r3) [inline annlite/storage/table.py:450]: remove the unused codes.

**Review by numb3r3 — CHANGES_REQUESTED** [review]

**Review by JoanFM — DISMISSED** [review]

File: annlite/storage/table.py [hunk 2]
```diff
@@ -281,18 +276,22 @@ def query(
...
             where_conds.append(where_clause)
-        where = ' and '.join(where_conds)
+            where = ' and '.join(where_conds)
 
         _order_by = f'{order_by or "_id"} {"ASC" if ascending else "DESC"}'
         _limit = f'LIMIT {limit}' if limit > 0 else ''
         _offset = f'OFFSET {offset}' if offset > 0 else ''
 
-        sql = f'SELECT _id from {self.name} WHERE {where} ORDER BY {_order_by} {_limit} {_offset}'
+        if len(where_conds) > 0 and where:
```
Comment (numb3r3) [inline annlite/storage/table.py:289]:
```suggestion
        if where:
```

File: annlite/storage/table.py [hunk 3]
```diff
@@ -308,7 +307,10 @@ def _offset_factory(_, record):
         cursor = self._conn.cursor()
 
         try:
-            offsets = cursor.execute(sql, params).fetchall()
+            if len(where_conds) > 0 and where:
```
Comment (numb3r3) [inline annlite/storage/table.py:310]:
```suggestion
            if where:
```

**Review by numb3r3 — CHANGES_REQUESTED** [review]

**Review by numb3r3 — APPROVED** [review]
LGTM

**Pull request merged**